docs/Search-version-list-resource.md

```diff
@@ -0,0 +1,49 @@
+# Search version list resource
+
+**Subsystem: Search 🔎**
+
+The version list resource is bookkeeping used to update the search subsytem. Effectively, this resource is a mapping of package IDs to their versions. It is initially created by the [Db2AzureSearch](../src/NuGet.Jobs.Db2AzureSearch) tool and is kept up-to-date by the [Catalog2AzureSearch](../src/NuGet.Jobs.Catalog2AzureSearch) job.
+
+## Purpose
+
+The [search index](./Azure-Search-indexes.md#Search-index) stores up to 4 documents per package ID. These documents store the metadata for the latest listed version across these pivots:
+
+1. Prerelease - Includes packages that aren't stable
+1. SemVer - Includes packages that require SemVer 2.0.0 support
```

Comment:
Did you mean "2." here?

Reply:
Markdown properly renders this: https://github.com/NuGet/NuGet.Services.Metadata/blob/b0a1dc0c2f6124be498845fb8e3c794da78a6af9/src/NuGet.Jobs.Auxiliary2AzureSearch/README.md#using-dev-resources

Using `1.` repeatedly lets you move items in the numbered list without needing to update its number. This trick makes it a little easier to maintain READMEs, but it does make the "raw" markdown a tiny bit less readable. I'm ok with this tradeoff given that most people will read this README on GitHub.

Reply:
Yeah I get the maintenance point here and agree that most people will read this on GitHub where it's correctly rendered 😃